sketch2Code.sketchplugin

```diff
@@ -103,9 +108,35 @@ function templateBinding(targetView)
 
           var layoutString = layoutDefinition(layer, layerName)
           [layouts addObject:"\t" + layoutString]
+          [allLayers addObject:layer]
```

Comment:
@csacsi I think, this should be outside of the if condition (above line 93), because there could be layers, for that we do not have defined mapping in the descriptors, therefore they would not be added to allLayers, and won't be hidden